tests/TileDB.CSharp.Test/DeletesTest.cs

```diff
@@ -0,0 +1,928 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TileDB.CSharp.Test
+{
+    [TestClass]
+    public class DeletesTest
+    {
+        private static void InitTest(Context ctx, string arrayName, LayoutType layout, bool duplicates)
+        {
+            Console.WriteLine(Assembly.GetAssembly(typeof(TileDB.CSharp.Array))!.FullName);
+            Console.WriteLine($"TileDB Core version: {TileDB.CSharp.CoreUtil.GetCoreLibVersion()}");
```

Comment:
Are these logs always needed to be printed in every test?

Also `Console.WriteLine` is not the right way to log in tests; MSTest seems to have the [`Logger`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.unittesting.logging.logger) class.